docs/testing.md

```diff
@@ -1,49 +1,119 @@
 # Narrative Unit Testing
 
-### Short version
-1. Install the Narrative and activate its virtualenv (if appropriate)  
+***Table of Contents***
+
+- [Installation](#installation)
+- [Short Version](#short-version)
+- [Long Version](#long-version)
+  - [About](#about)
+  - [How to Run Tests](#how-to-run-tests)
+  - [Add Credentials for Tests](#add-credentials-for-tests)
+  - [Testing with Travis-CI and Coveralls](#testing-with-travis-ci-and-coveralls)
+  - [Adding Your Own Tests](#adding-your-own-tests)
+  - [Manual Testing and Debugging](#manual-testing-and-debugging)
+
+## Installation
+
+Install grunt-cli via the following command:
+
+```
+npm install -g grunt-cli
+```
+
+## Short version
+
+1. Install the Narrative and activate its conda environment (if appropriate)
```

Comment:
maybe just `activate its environment`? It doesn't HAVE to be conda, though that's what the default instructions give. I'm sure it'll work fine with venv, or poetry, or pipenv, or whatever.